onadata/apps/api/viewsets/attachment_viewset.py

```diff
@@ -83,6 +84,16 @@ def retrieve(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    @action(methods=['GET'], detail=False)
+    def count(self, request, *args, **kwargs):
+        response = {}
+        xform = request.query_params.get('xform')
+        total_count = Attachment.objects.filter(
```

Comment:
I think it is best we rely on using `self.filter_queryset(self.get_queryset())` so that we maintain the queryset and filters to applied to it by the viewset. Hence
```python
data = {
  "count":  self.filter_queryset(self.get_queryset()).count()
}

return Response(data=data)
```